core/src/main/resources/hudson/security/GlobalSecurityConfiguration/index.groovy

```diff
@@ -11,12 +11,16 @@ def f=namespace(lib.FormTagLib)
 def l=namespace(lib.LayoutTagLib)
 def st=namespace("jelly:stapler")
 
-l.layout(permission:app.ADMINISTER, title:my.displayName, cssclass:request.getParameter('decorate')) {
+l.layout(permission:app.SYSTEM_READ, title:my.displayName, cssclass:request.getParameter('decorate')) {
     l.main_panel {
         h1 {
             l.icon(class: 'icon-secure icon-xlg')
             text(my.displayName)
         }
+        if (!h.hasPermission(app.ADMINISTER)) {
+            set("readOnlyMode", "true")
+            input(id: "readOnlyMode", type: "hidden", name: "readOnlyMode", value: "true")
```

Comment:
Ideally this should be solved differently. Just have the plugins generate an element they want to check in JS, based on the variable being set?

Reply:
sure that would work too, I just thought of this first

Reply:
Just seems like a hack. So far the `readOnlyMode` variable was sufficient for core, it would be nice if we kept that (until we can't anymore).

Reply:
done